deployment/helm/ditto/templates/thingssearch-deployment.yaml

```diff
@@ -254,8 +254,6 @@ spec:
               value: "{{ .Values.thingsSearch.config.updater.backgroundSync.throttle.throughput }}"
             - name: BACKGROUND_SYCN_THROTTLE_PERIOD
               value: "{{ .Values.thingsSearch.config.updater.backgroundSync.throttle.period }}"
-            - name: DITTO_POLICIES_ENFORCER_CACHE_MAX_SIZE
-              value: "{{ .Values.thingsSearch.config.updater.stream.policiesEnforcer.cache.maxSize }}"
             - name: DITTO_POLICIES_ENFORCER_CACHE_EXPIRE_AFTER_WRITE
```

Comment:
@kalinkostashki shouldn't those keys then also be deleted?

Reply:
yes, you're right I forgot about them.
will remove them as well :)